src/utils/theme.js

```diff
@@ -0,0 +1,32 @@
+import { DefaultTheme } from 'react-native-paper'
+
+const theme = {
+  ...DefaultTheme,
+  colors: {
+    ...DefaultTheme.colors,
+    background: '#FFF',
+    selected: '#309D9C',
+    favorited: '#DA71B7',
+    error: '#FF0C3E',
+  },
+  fonts: {
```

Comment:
I'm not sure how this is going to work for `italic` and `bold` text.

**Note:** This will only use those fonts in `react-native-paper` or `react-navgiation` components. If you use the `Text` component directly from `react-native` it won't automatically use this font.

Reply:
Could you add records for `bold` and `italic`? Just map `bold` to the same `bcsans-bold` as `medium`, and then `italic` could just go to the regular `bcsans` if we aren't provided with an italic version? Is that how this works?

Reply:
I don't think that's how it works because this:

<img width="620" alt="Screen Shot 2020-10-06 at 14 34 16" src="https://user-images.githubusercontent.com/15054821/95262619-4c0eb380-07e1-11eb-81cb-811a2ffced1c.png">

renders differently then this:

<img width="575" alt="Screen Shot 2020-10-06 at 14 35 35" src="https://user-images.githubusercontent.com/15054821/95262643-592ba280-07e1-11eb-93b3-b26749455f47.png">

<kbd><img src="https://user-images.githubusercontent.com/15054821/95262630-53ce5800-07e1-11eb-8d79-71942df9569f.png" width="245" /></kbd> <kbd><img src="https://user-images.githubusercontent.com/15054821/95262569-3c8f6a80-07e1-11eb-9181-7b95b476af19.png" width="245" /></kbd>

It's subtle but there is a difference.

Reply:
Oh okay, that was just a guess.

Looking at that further though, does `<Text>Coming Soon!</Text>` render the same as `<Text style={{ fontFamily: 'bcsans' }}>Coming Soon!</Text>`? The docs make it look like the config needs to be within a `default` object, as well as be passed to the `configureFonts` function.

```js
const theme = {
  ...
  fonts: configureFonts({
    default: {
      regular...
      ...
    }
  })
}
```

Also, is that `Text` being imported from `react-native` or paper? As you mention it won't work with the one from `react-native`. Sorry if these are things you've already tried or are obvious. I'm not running the code, just reading it.

Reply:
Yes I tried that. Given we have a short timeline on this and my previous implementation was functional I suggest we stick to setting the regular font to `bcsans` in this way but then just use `styled-components` to set the `font-family` to `bcsans-bold`, `bcsans-italic` or `bcsans-bold-italic`. Unless you want to work on this?